.github/workflows/bump-license-year.yml

```diff
@@ -0,0 +1,37 @@
+#############################################################################
+# GitHub Action to bump license year
+#
+# Workflow starts every new year.
+#
+#############################################################################
+name: "Bump license year"
+on:
+  schedule:
+    - cron: '0 8 1 1 *'
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  bump:
+    name: Bump license year
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set Current Year
+        id: CURRENT_YEAR
+        run: |
+          echo "::set-output name=year::$(date +'%Y')"
+      - name: Modify Files
+        run: |
+          ./.ci/bump-license-year.sh $(expr ${{ env.YEAR }} - 1) ${{ env.YEAR }} .
+        env:
+          YEAR: ${{ steps.CURRENT_YEAR.outputs.year }}
+      - name: Push commit
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+          git commit -am "minor: Bump year to ${{ env.YEAR }}"
+          git push
```

Comment:
Add /n

Reply:
At first, there was /n already. Do you want me to revert back to https://github.com/checkstyle/sonar-checkstyle/compare/e9f34296a8ad2d417e49b2f0e03ec2c9c47c7528..1e2351f72a6d24f311e7b7786efe8423494dd212 ?

Reply:
Please get rid of red sign that GitHub shows you

Reply:
Changes Done